recipe/meta.yaml

```diff
@@ -30,7 +40,11 @@ requirements:
     - make                               # [linux]
   host:
     - zlib  # [unix]
-
+    - freetype  # [tk_variant == 'xft']
+    - fontconfig  # [tk_variant == 'xft']
+    - xorg-libxft  # [tk_variant == 'xft']
```

Comment:
please consider converting other xorg  CDTs into standard conda packages as well

Reply:
That needs to be a bigger discussion I guess about adding more dependencies to

Reply:
Actually, I just opened https://github.com/conda-forge/cdt-builds/pull/68 to get a libXft-devel cdt.  Is the team moving away from cdt's, or is there just a concern here for tk, as a dependency of python?